src/Program.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using McMaster.Extensions.CommandLineUtils;
+using Newtonsoft.Json;
+
+namespace secondtry
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+           commands obj = new commands();
+           
+           obj.Idel(args);
+        }
+    }
+
+    class commands
+    {
+        public int Idel(string [] commands)
+        {
+            var app = new CommandLineApplication();
+            var helptemplate = "-h|--help";
+            app.HelpOption(helptemplate);
+            app.Command("edit", e => {
+            e.HelpOption(helptemplate);
+            e.Description = "Dieser Befehl soll es ermöglichen die hinterlegte Konfiguration zu editieren.";
```

Comment:
Willst du das hier wirklich? Aber ich glaube darüber haben wir schonmal geredet. Die Konfiguration kann der User in jedem beliebigen Editor anpassen. Das toll hier soll erstmal nur verarbeiten.

Reply:
Ne das muss ich noch ändern, hatte es aber noch drin gelassen